### DISCUSSION TITLE
"Your device is likey to fail soon"-Warning frequently on pretty new USB

### DISCUSSION THREAD

**Discussion by elisenlebkuch** [ventoy/Ventoy#2776] [Q&A]
I get the "Your device is likely to fail soon"-warning on all kinds of linux ISOs or when I plug it into a linux system to put ISOs onto it. The Drive is pretty new (a little over a year old now), but this isn't a new discovery for me, I've been getting these warnings since I put ventoy on it a little less than a year ago. Is this normal for ventoy drives?

**Comment by steve6375** [comment]
Depends on what the device is and how you have been using it and how often you have been using it.
It's like saying 'my car's engine warning light is on - is this normal'! without knowing how old the car is, what make/model it is, it's service history, mileage and how well it has been treated.

**Reply by elisenlebkuch** [reply]
SunDisk Extreme PRO 128GB about 1 year and 4 months old. I use it maybe 2-3 times per month.

**Comment by steve6375** [comment]
I have about 4 of those which I use for multibooting all the time. Never had a warning or a fail.
What exactly is giving you this warning? Maybe it is more to do with your OS or hardware?
They dont have SMART, so how does it detect that it is failing?

**Reply by elisenlebkuch** [reply]
I don't know how the warning is generated, hence the question. I only know that on some ISOs it appears. It is a simple notification, nothing special.
